### ISSUE TITLE
Hint box spacing

### ISSUE THREAD

**Issue by Sepitus-exe** [Sepitus-exe/WKElementaryDark#27]
![image](https://user-images.githubusercontent.com/82216846/213050925-a182c97d-aa1a-4b07-a279-b1ad1ee16e14.png)

Inherent WK issue, possibly fixable on our end by altering the `#item-info blockquote` selector

Default WK version
```
#item-info blockquote {
  margin: 1em 0 0;
  padding: 10px;
  background-color: #fbfbfb;
  color: #888888;
}
```

**Comment by wrex** [comment]
@Sepitus-exe: This is how it renders on my laptop with the current code:

<img width="1159" alt="Screen Shot 2023-01-19 at 9 35 54 AM" src="https://user-images.githubusercontent.com/653208/213518876-4b5abe1a-3290-4e5e-bac7-922c753a1cdf.png">

Is this issue already resolved?

**Comment by Sepitus-exe** [comment]
It seems to only occur on untraditional resolutions and only in reviews/lessons. There is a way to increase the line spacing, which would solve the issue, but doing so might negatively affect the looks on the intended resolutions.

![obrazek](https://user-images.githubusercontent.com/82216846/213791275-c4a4efc5-5cb0-4f28-baf0-665a4548397d.png)
![obrazek](https://user-images.githubusercontent.com/82216846/213791350-015917d8-0303-448c-96f4-73b34742d117.png)

The issue is going to be moved into the wontfix category, but might be revisited in the future.

**Comment by Sepitus-exe** [comment]
I will probably leave it open for a little while longer, since the default spacing doesn't feel pleasant. It might be for the better to increase it a bit.

**Comment by Sepitus-exe** [comment]
Yeah, I will leave this as wont fix, since it feels too insignificant to worry about
